_episodes/04-formatting.md

```diff
@@ -433,4 +434,55 @@ You can add a drop shadow effect to images by applying the
 
 [jekyll-link-tag]: https://jekyllrb.com/docs/liquid/tags/#link
 
+## Adding Formatted Equations
+
+The template supports rendering of equations via [KaTeX](https://katex.org/).
+This option must be activated by adding `math: true` to the YAML front matter
+of the Markdown file where you wish to use it.
+
+Mathematical expressions can then be added to the page content using the LaTeX syntax.
+
+Expressions can be written inline:
+
+~~~
+{% raw %}Inline expressions can be added between `$` symbols, e.g. $E = Mc^2$.{% endraw %}
+~~~
+{: .source}
+
+with the result:
+
+Inline expressions can be added between `$` symbols, e.g. $E = mc^2$.
+
+Or as a block across multiple lines:
+
+~~~
+{% raw %}$$
+    \lim_{x \rightarrow 0}
+    \frac{
+        \sin x
+    } {
+        x
+    }
+    = 1
+$${% endraw %}
+~~~
+{: .source}
+
+with the result:
+
+$$
+    \lim_{x \rightarrow 0}
+    \frac{
+        \sin x
+    } {
+        x
+    }
+    = 1
+$$
+
+The example above was taken from the chapter _Typesetting Mathematical Formulae_,
+in [The Not So Short Introduction to LaTeX](https://tobi.oetiker.ch/lshort/lshort.pdf),
```

Comment:
Could we use a more accessible source? PDFs are not great for folks who use screen readers. 

The only drawback is that a lot of the ready-made demos that exist are irritatingly complex equations (i.e. https://forever-jekyll.github.io/katex/demo/2021/07/08/katex-demo/)

Reply:
Good point re: (in)accessibility of PDFs. The other resource I was recommended was https://en.wikibooks.org/wiki/LaTeX/Mathematics. I was hoping for something a bit easier to navigate TBH!

Reply:
Looking again at https://en.wikibooks.org/wiki/LaTeX/Mathematics and I think I have come around to it, as a nice balance of enough information without being as overwhelming as many of the other references I have been pointed towards. I have added a link to this, but kept the PDF link too, because I would like to credit the source for the example I used.